### DISCUSSION TITLE
Swissup modules are not working with  nginx version: nginx/1.19.6

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#243] [1. Help]
Hello @vovayatsyuk 

We are enabling whole swissup theme on server from our staging server we have tried for several hours but we were not able to activate staging code to live server for one of our client.

Live server was throwing http 500 error while enabling swissup core , breeze, navigationpro, pagespeed etc and on our staging server every module is working fine which is having nginx version => nginx/1.25.1. and our live server nginx version => nginx/1.19.6

Kindly guide us what we can take major steps for the same.

It's urgent please reply as soon as possible @vovayatsyuk.

Thanks in advance.
JamesDenish

**Comment by vovayatsyuk** [comment]
I can't help you without knowing the error message. Can you check your Nginx error logs?

**Comment by JamesDenish** [comment]
Hello @vovayatsyuk 

We have found some server logs error to be like below:
PHP message: PHP Fatal error:  Cannot declare class Swissup\\Breeze\\Model\\Filter\\Dom\\FormValidation, because the name is already in use in /home/www/coplax/htdocs/app/code/Swissup/Breeze/Model/Filter/Dom/PreloadCriticalImages.php on line 7
when we are enabling module swissup_Breeze it is throwing 500 error and logs is showing above error.

Please reply it's urgent.

Thanks in advance
JamesDenish

**Reply by vovayatsyuk** [reply]
Looks like you installed the Breeze module in two places. Remove it from the app/code and leave the one from vendor/swissup.

**Comment by JamesDenish** [comment]
Hello @vovayatsyuk 

We don't have swissup folder in vendor and we have only placed in app/code only.

Thanks in advance

**Reply by vovayatsyuk** [reply]
I don't know what to say. It's definitely not related to Nginx. Did you edit the Breeze source code? If there is no duplication of classes outside of app/code folder, then you need to search inside the app/code folder.
